pubspec.yaml

```diff
@@ -60,7 +60,7 @@ dependencies:
   html_editor_enhanced:
     git:
       url: https://github.com/linagora/html-editor-enhanced.git
-      ref: main
+      ref: bugfix/cannot-break-quote-when-press-enter-key
```

Comment:
I found the change maybe conflict with double scroll bars. please demo with some other browser to make sure it work

Reply:
Tested on several browsers (`Chrome, Edge, Opera, Firefox, Safari`), it works fine. Please see below demo video:


https://github.com/user-attachments/assets/59a6f650-fbb3-4255-9110-698e18b6c355